arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -486,7 +479,7 @@
         "tags": [
           "Operations"
         ],
-        "operationId": "getAvailableOperations",
+        "operationId": "Workspace_listOperations",
```

Comment:
So AutoRest threw an error if this wasn't renamed to either "list" or "list by subscription ID" - changing this fixed it